ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -585,8 +585,8 @@ GetTFToNgOpMap() {
       {"AddV2", {std::make_shared<opset::Add>()}},
       {"Any", {std::make_shared<opset::ReduceLogicalOr>(), constant}},
       {"All", {std::make_shared<opset::ReduceLogicalAnd>(), constant}},
-      {"ArgMax", {std::make_shared<ngraph::op::ArgMax>()}},
-      {"ArgMin", {std::make_shared<ngraph::op::ArgMin>()}},
+      {"ArgMax", {std::make_shared<opset::Squeeze>(), constant}},
+      {"ArgMin", {std::make_shared<opset::Squeeze>(), constant}},
```

Comment:
Add `opset::TopK` here.

Reply:
Yea I thought so too but NGRAPH_TF_LOG_PLACEMENT only shows Squeeze and constant.